queries/idl/highlights.scm

```diff
@@ -8,19 +8,44 @@
   "bitmask"
   "bitset"
   "@annotation"
-  "interface"
   "exception"
   "typedef"
+  "home"
+  "typeid"
+  "typeprefix"
+  (interface_kind)
+  (value_kind)
+  "component"
+  "porttype"
+  "connector"
+  "eventtype"
+  "valuetype"
 ] @keyword.type
 
+(import_dcl
+  "import" @keyword.directive)
```

Comment:
Shouldn't this be `@keyword.import`, looking like it's part of import declaration